lib/use_cases/send_slack_messages.rb

```diff
@@ -12,27 +12,39 @@ def initialize(
       @message_presenter = message_presenter
     end
 
-    def execute
-      open_pull_requests = pull_request_gateway.execute
-      teams = team_gateway.execute
+    def execute(team: nil)
```

Comment:
Feels weird to have potentially hidden behavior by passing in nothing vs passing in the team? Not sure though

Reply:
I would expect send slack messages to send slack messages, whereas passing in a team is more like, "send slack messages for team"